merlin/metaanalysis/aggregatedata.py

```diff
@@ -0,0 +1,55 @@
+from merlin.core import analysistask
+import pandas
+
+
+class AggregateData(analysistask.AnalysisTask):
+    """
+    A metaanalysis task that aggregates data from multiple datasets
+    """
+    def __init__(self, metaDataSet, parameters=None, analysisName=None):
+        super().__init__(metaDataSet, parameters, analysisName)
+
+        if 'overwrite' not in self.parameters:
+            self.parameters['overwrite'] = False
+
+        self.metaDataSet = metaDataSet
+
```

Comment:
It would be good to add a function to AnalysisTask called get_supported_datasets that returns a list of the types of datasets that the analysis task can run on and raise an error when anyone tries to run it on the wrong dataset type. This should help prevent people from trying to run an analysis task on the wrong kind of dataset.